src/HeuristicCompletion-Model/CoASTResultSetBuilder.class.st

```diff
@@ -110,6 +110,11 @@ CoASTResultSetBuilder >> visitGlobalNode: aRBVariableNode [
 	^ self visitVariableNode: aRBVariableNode
 ]
 
+{ #category : #visiting }
```

Comment:
Is this method part of this PR?

Reply:
I ran into a DNU and decided to just fix it as part of this PR.

 (the code completion re-implements the visitor without using the Trait or inheriting, thus the backward compatibility method put in place is not helping here...)